codechefcli/utils/helpers.py

```diff
@@ -52,6 +52,20 @@ def request(session, method, url, **kwargs):
         sys.exit(1)
 
 
+def sort_data_rows(data_rows, sort):
```

Comment:
Please document this method. Seems like it is not returning anything as well. This is not the correct use case. Have you tested this method?

Reply:
Sir this method was from before the decroator method....i forgot to remove it.